static/data-ingestion.yaml

```
@@ -169,93 +104,178 @@ paths:
-        Create a new Product entity in given `sku` and `scopes`. If product exists for the same SKU and scopes item will be ignored.
-        Suitable only for onboarding phase
-      operationId: ProductsPost
+        - metadata
+      summary: Product Attributes Metadata Patch API
+      description: Product attributes patch request allows you to update registered Product Attribute Metadata
+      operationId: ProductMetadataPatch
       parameters:
-        - $ref: '#/components/parameters/DATA_SPACE_ID'
-        - $ref: '#/components/parameters/ContentType'
-        - $ref: '#/components/parameters/xApiKey'
-        - $ref: '#/components/parameters/xGwSignature'
-        - $ref: '#/components/parameters/ContentEncoding'
+        - name: DATA_SPACE_ID
+          in: path
+          description: Data Space ID
+          required: true
+          style: simple
+          schema:
+            type: string
+        - name: Content-Type
+          in: header
+          required: true
+          schema:
+            type: string
+            enum: [ application/json ]
+        - name: x-api-key
+          in: header
+          required: true
+          schema:
+            type: string
+          description: Production public API key
+        - name: x-gw-signature
+          in: header
+          required: true
+          schema:
+            type: string
+          description: JWT generated for Production public API key
+        - name: Content-Encoding
+          in: header
+          required: false
+          schema:
+            type: string
+            enum: [ gzip ]
+          description: Use this header if the payload is compressed with gzip
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/FeedMetadata'
+            examples:
+              FeedWithMetadataInformation:
+                summary: Example of product attributes metadata feed
+                description: Enables the feed with product attribute information that needs to be updated
+                value:
+                  [
+                    {
+                      "code": "name",
+                      "scope": {
+                        "locale": "En"
+                      },
+                      "label": "Updated - Product Name",
+                      "dataType": "SELECT",
+                      "roles": [ "PDP", "PLP" ],
+                      "filterable": true,
+                      "sortable": true,
+                      "searchable": true,
+                      "searchWeight": 77,
+                      "searchTypes": [ "STARTS_WITH", "AUTOCOMPLETE" ]
+                    }
+                  ]
       responses:
-        '202':
-          $ref: '#/components/responses/AcceptedResponse'
+        '200':
+          description: Request accepted. Check the "invalidFeedItems" node for any invalid items that were not consumed
```

for CCDM API only request with all valid items is processed. 
```suggestion
          description: All items in the request are accepted for further processing
```

---

Done